packages/p2p/src/components/my-ads/ad-status.scss

```diff
@@ -1,46 +1,38 @@
+@mixin ad-status-base($background-color) {
+    &:before {
+        content: '';
+        height: 100%;
+        width: 100%;
+        opacity: 0.16;
+        display: block;
+        position: absolute;
+        left: 0;
+        top: 0;
+        border-radius: 1.6rem;
+        background-color: $background-color;
+    }
+    align-items: center;
```

Comment:
Added `align-items: center` and `justify-content: center` to center the text for active and inactive ad status

### Before
![Screenshot 2022-08-01 at 11 28 00 AM](https://user-images.githubusercontent.com/103016120/182068058-ddfac82c-8a71-4069-9d2d-c1283df20431.png)
![Screenshot 2022-08-01 at 11 36 05 AM](https://user-images.githubusercontent.com/103016120/182068069-a6932527-0237-4087-80e4-e36f0cc62ffe.png)

### After
![Screenshot 2022-08-01 at 11 36 08 AM](https://user-images.githubusercontent.com/103016120/182068083-7b10b82c-e247-445a-80df-52eec0405875.png)

![Screenshot 2022-08-01 at 11 44 15 AM](https://user-images.githubusercontent.com/103016120/182068126-431b7b2d-9ae0-42ca-937e-bbde63a9b22e.png)